core-java-modules/pom.xml

```diff
@@ -118,6 +118,9 @@
         <module>core-java-regex-2</module>
         <module>core-java-uuid</module>
         <module>pre-jpms</module>
+        <module>java-collections-conversions</module>
```

Comment:
Can you insert these new modules alphabetically in this list?

Reply:
Addressed